### ISSUE TITLE
publish the official versions on a open website

### ISSUE THREAD

**Issue by ThierryO** [inbo/protocolsource#3]
The source code is public. So why don't we make the rendered versions public too?

**Comment by hansvancalster** [comment]
Yes, that is the idea. But for now the repository is very much a work in progress. We first have to be happy about every aspect related to management of the protocols before we actually start to add content of protocols. 

Here is a checklist of what needs to be done:

- [ ] agree on folder structure
- [ ] agree on branching model
- [ ] use bookdown instead of single Rmarkdown for template to more easily allow project-specific protocols to be derived from generic protocols
- [ ] consistently use English language for folders, function names, etc. But content of protocols will be Dutch
- [ ] setup a `protocols-helper` R package with helper functions. 
- [ ] add contributing guidelines
- [ ] convert existing protocols (in Word format) to Rmarkdown and add them

**Comment by hansvancalster** [comment]
@ThierryO 
I think the next step is to prepare a `protocols-website` repo and link this to this repo via Travis CI or Github runners? 
I think this would need to be done first, before we can start migrating old-style docx protocols to bookdown protocols (one by one in separate PR)? Well, maybe migration can start already, but merging the PRs will need to wait until this infrastructure is ready.

**Comment by hansvancalster** [comment]
To be discussed steps to prepare for publications of the protocols:

- [ ] create protocols-website repo (just an empty repo? need to create gh-pages branch? settings?)
- [ ] create a GitHub action in this repo. The action should be triggered on release. It can be based on largely on https://github.com/inbo/protocols/blob/master/.github/workflows/generate-artifact-pr.yml but with the following differences
     - should run protocolhelper:::render_release() instead of protocolhelper::render_protocol()
     - should somehow add, commit and push all files in publish folder to the protocols-website repo

(see also related https://github.com/inbo/protocolhelper/issues/17)

Or can this be done better with a Dockerfile?

**Comment by ThierryO** [comment]
A Docker file is much more efficient as it has all (or at least most) dependencies installed. Downloading the Docker is faster than installing all the packages from scratch.

Note that we often exceed our artifact storage quota. So you shouldn't rely on artifacts.

**Comment by hansvancalster** [comment]
A first protocol has been published on https://inbo.github.io/protocols/, so closing this issue
